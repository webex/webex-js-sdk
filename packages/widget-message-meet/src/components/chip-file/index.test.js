import React from 'react';
import renderer from 'react-test-renderer';

import ChipFile from '.';

describe(`ChipFile component`, () => {
  const props = {
    name: `testFile.jpg`,
    size: `123 KB`,
    thumbnail: `blob:localhost/testFile.jpg`,
    type: `image`
  };
  const component = renderer.create(<ChipFile {...props} />);

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
