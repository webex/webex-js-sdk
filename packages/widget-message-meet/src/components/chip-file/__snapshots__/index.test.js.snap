exports[`ChipFile component renders properly 1`] = `
<div
  className="chip">
  <div
    className="file-thumbnail">
    <img
      role="presentation"
      src="blob:localhost/testFile.jpg" />
  </div>
  <div
    className="file-info">
    <div
      className="file-name">
      testFile.jpg
    </div>
    <div
      className="file-meta">
      <div
        className="file-size">
        123 KB
      </div>
      <div
        className="file-type">
        image
      </div>
    </div>
  </div>
  <div
    className="chip-action">
    <button
      className="button"
      onClick={[Function]}
      onKeyUp={[Function]}>
      <div
        className="icon"
        title={undefined} />
    </button>
  </div>
</div>
`;
